pkgs/development/guile-modules/guile-xcb/default.nix

```diff
@@ -26,10 +26,12 @@ stdenv.mkDerivation rec {
   ];
 
   configureFlags = [
-    "--with-guile-site-dir=$out/share/guile/site"
-    "--with-guile-site-ccache-dir=$out/share/guile/site"
+    "--with-guile-site-dir=\${out}/share/guile/site"
```

Comment:
you can also use `placeholder` for this.

also: please adjust the commit message to contributing.md

Reply:
I don't quite get the idea with a placeholder - @out@ doesn't get replaced automatically.

Reply:
```suggestion
    "--with-guile-site-dir=${placeholder "out"}/share/guile/site"
```

is what I meant

Reply:
Aha, got the idea. Now I'm deciding which one is less clumsy :)